### ISSUE TITLE
fix validation memory leak FBPConvNet_train.py

### ISSUE THREAD

**Issue by hyt35** [CambridgeCIA/LION#115]
In `example_scripts/FBPConvnet_train.py`: unnecessary memory usage during validation step, going from 11gb to 18gb. Please use with `torch.no_grad()` flag when computing validation loss (right after `model.eval()` around line 100 should work)

**Comment by AnderBiguri** [comment]
Indeed! The real solution is that this script will be changed to use `SupervisedSolver` which should be doing that. But indeed this is a bug :)
